sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/MetadataType.java

```diff
@@ -0,0 +1,81 @@
+package com.smartdevicelink.proxy.rpc.enums;
+
+/**
+ * Defines the metadata types that can be applied to text fields
+ *
+ */
+public enum MetadataType {
+
+	/**
+	 * The data in this field contains the title of the currently playing audio track.
+	 */
+	mediaTitle,
```

Comment:
All enums should be in all caps. Since the string value is different than all caps, you will have to add a variable to hold that. See [HMILevel](https://github.com/smartdevicelink/sdl_android/blob/master/sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/HMILevel.java) where the string value is different than the enum name.